src/Common/Version.props

```diff
@@ -0,0 +1,21 @@
+<Project>
+  <PropertyGroup>
+    <AssemblyVersion>1.11.1</AssemblyVersion>
```

Comment:
I think we don't need to keep those two projects in sync. We might need to update Nunit only when we change something there or there is a breaking change in PW.

Reply:
Yes, that was my thought too, *but* it has a dependency on Playwright. While it's technically a `>=` dependency, it felt like keeping them in-sync would be the right bet.